### PR TITLE
Install scripts

### PR DESCRIPTION
**Pull request by rs028**
I made a few adjustments to the install scripts. Other than that, the installation procedure looks all right. I have updated the wiki pages, accordingly.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     - ./tools/install/install_openlibm.sh $PWD
 
   install:
-    - ./tools/build.sh ./mcm_subset.fac
+    - ./tools/build.sh ./tools/mcm_subset.fac
```
Comment (spco) [inline .travis.yml]: Should be `mcm_example.fac`! :)

Comment (rs028) [inline .travis.yml]: Of course :)

**Comment by spco** [comment]
Just give me a nudge to merge when you're happy.

**Comment by rs028** [comment]
Good to go.

**Comment by spco** [comment]
Just tidied up the history, no other changes.

**Pull request merged**